src/SonarLint.Test.props

```diff
@@ -26,5 +26,12 @@
     <PackageReference Include="FluentAssertions" Version="5.9.0" />
     <PackageReference Include="FluentAssertions.Analyzers" Version="0.11.4" />
   </ItemGroup>
+  
+  <ItemGroup>
```

Comment:
For the entire project (production + test code), we have also this:
https://github.com/SonarSource/sonar-dotnet/blob/3622c4a4126173fd74812d205e4e490fbddb548a/analyzers/Analyzers.targets#L16-L24

Can you add it too, maybe in a separate PR? All except the immutable collections seems relevant in SL project

Reply:
The least used is `System.Diagnostics`. The purpose of this one is to be able easily do `Debugger.Break()` or `Debugger.Launch` anywhere

Reply:
https://github.com/SonarSource/sonarlint-visualstudio/issues/5426